src/plugins/intel_gpu/src/graph/impls/ocl/primitive_base.hpp

```diff
@@ -88,11 +91,14 @@ struct typed_primitive_impl_ocl : public typed_primitive_impl<PType> {
     static std::unique_ptr<primitive_impl> create(const typed_program_node<PType>& arg, const kernel_impl_params& impl_param) {
         // concat buffer fusing for dynamic shape is adaptively applied at runtime. So we need to build dynamic impl at build time.
         if (impl_param.can_be_optimized() &&
-            !((impl_param.is_type<concatenation>() ||
+            !((impl_param.is_type<concatenation>() || // Runtime skippable?
                impl_param.is_type<gather>() ||
                impl_param.is_type<permute>() ||
                impl_param.is_type<strided_slice>() ||
                impl_param.is_type<broadcast>() ||
+               impl_param.is_type<scatter_elements_update>() ||
+               impl_param.is_type<scatter_nd_update>() ||
+               impl_param.is_type<scatter_update>() ||
```

Comment:
yes we can replace this as 
if (impl_param.can_be_optimized() &&( runtime_skippable() || is_type<crop>())

Reply:
Fixed it. thanks.